homework-5/Homework-5.playground/Contents.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+var country: [String: String]
+country = [
+    "Germany": "Western Europe. The surname Hitler is banned in Germany",
+    "Italy": "Southern Europe. The ending of all words in Italian only with a vowel",
+    "France": "Western Europe. France is the birthplace of the bicycle",
+]
+
+country["Italy"] = nil
+country["Spain"] = "located on the Iberian Peninsula. Football is considered the national sport"
+country["Germany"] = "The first book was printed in Germany"
+country["France"] = "France is the birthplace of cinema"
+print(country)
+
+var city: [String]
+city = [
+    "Amsterdam",
+    "Copenhagen",
+    "Vilnius",
+]
```

Comment:
We need to create a dictionary with more than 6 letters countries from the initial dictionary. It means we need to filter the elements where country name is more than 6 letters long